tests/test_double_blind_conference.py

```diff
@@ -1334,6 +1334,36 @@ def test_consoles(self, client, test_client, selenium, request_page):
         assert 'Status' == tabs.find_element_by_id('paper-status').find_element_by_class_name('row-4').text
         assert 'Decision' == tabs.find_element_by_id('paper-status').find_element_by_class_name('row-5').text
 
+    def test_reassign_reviewers_with_conflict_of_interest(self,client,test_client,selenium,request_page):
+        builder=openreview.conference.ConferenceBuilder(client)
+        builder.set_conference_id('AKBC.ws/2019/Conference')
+        builder.set_submission_stage(double_blind = True, public = True)
+        builder.set_conference_short_name('AKBC 2019')
+        builder.has_area_chairs(True)
+        builder.enable_reviewer_reassignment(True)#enable review reassignment so that the assign_Reviewer_Textbox is rendered on page
+        conference = builder.get_result()
+
+        #area chair to reassign reviewer
+        ac_client=openreview.Client(baseurl = 'http://localhost:3000', username='ac@mail.com', password='1234')
+        request_page(selenium,"http://localhost:3000/group?id=AKBC.ws/2019/Conference/Area_Chairs",ac_client.token)
+
+        show_Reviewers_AnchorTag=selenium.find_element_by_xpath('//*[@id="1-reviewer-progress"]/a')
+        selenium.execute_script("arguments[0].click()",show_Reviewers_AnchorTag) #click on "show reviewers anchor tag"
+        time.sleep(5) #leave some time for the API and js to complete
+
+        #get the reviewers textbox and click on it to show dropdown
+        assign_Reviewer_Textbox_Present=EC.presence_of_element_located((By.XPATH,'//*[@id="1-add-reviewer"]/div/input'))
+        WebDriverWait(selenium,5).until(assign_Reviewer_Textbox_Present)
```

Comment:
What is the difference between WebDriverWait and time.sleep? When should we use one vs. another?

Reply:
i think the timeout defined in webdriverwait is the max time that it will wait and the actual waiting time might be shorter if the specified element is generated in dom quicker.